concourse/pipelines/templates/build_pipeline-tpl.yml

```
@@ -479,6 +528,14 @@ resources:
     bucket: data-gpdb-ud-pxf-build
     json_key: ((pxf-storage-service-account-key))
     versioned_file: ((pxf-build-bucket-prefix))/snapshots/pxf-gp[[gp_ver]]-ubuntu18.04.tar.gz
+
+- name: pxf_gp[[gp_ver]]_tarball_rhel8
+  type: gcs
+  icon: google-drive
+  source:
+    bucket: data-gpdb-ud-pxf-build
+    json_key: ((pxf-storage-service-account-key))
+    versioned_file: ((pxf-build-bucket-prefix))/snapshots/pxf-gp[[gp_ver]]-rhel8.04.tar.gz
```

```suggestion
    versioned_file: ((pxf-build-bucket-prefix))/snapshots/pxf-gp[[gp_ver]]-el8.tar.gz
```
this would seem to fit the pattern, is this how it is produced ?

---

It is pushed up to GCP in a put step and the param for the put does correctly match the `.el8` convention. I will update it accordingly